tasks/workflow/worker.go

```diff
@@ -227,7 +227,7 @@ func (w *worker) handleExecution(t *taskExecution) {
 		err = errors.Errorf("Unknown TaskType %d (%s) for TaskExecution with id %q", t.taskType, t.taskType.String(), t.taskID)
 	}
 	if err != nil {
-		events.WithContextOptionalFields(ctx).NewLogEntry(events.LogLevelERROR, t.targetID).Registerf("%v", err)
+		events.WithContextOptionalFields(ctx).NewLogEntry(events.LogLevelDEBUG, t.targetID).Registerf("%v", err)
```

Comment:
Couldn't we keep the log level to ERROR here, and have instead a change in `helper/sshutil/sshutil.go` RunCommand() function
to have RunCommand() log at debug level the value of errors.WithStack(err) but have this function RunCommand() return directly err.
As err comes from an error running a command, we could just return this error in RunCommand() without the stack trace

Reply:
Yes, I thought first it was the solution but I didn't see any change when I tested it except we loose first stack trace steps.

Reply:
Maybe it can be useful to keep it anyway

Reply:
What will be the resulting logs if we have something like this?:
```suggestion
		events.WithContextOptionalFields(ctx).NewLogEntry(events.LogLevelERROR, t.targetID).Registerf("%v", err)
		events.WithContextOptionalFields(ctx).NewLogEntry(events.LogLevelDEBUG, t.targetID).Registerf("%+v", err)
```

Reply:
Same logs with "%v" and "%+v"

Reply:
But why???? :scream:

Reply:
found the problem here: https://github.com/ystia/yorc/blob/bugfix/GH-503-display-stacktrace-in-debug/tasks/workflow/worker.go#L744